catalog/ui/search-ui/standard/src/main/webapp/js/controllers/cesium.controller.js

```diff
@@ -20,27 +21,29 @@ define(['underscore',
         'properties',
         'js/view/cesium.metacard',
         'jquery',
-        'drawHelper'
-    ], function (_, Marionette, Cesium, Q, wreqr, properties, CesiumMetacard, $, DrawHelper) {
+        'drawHelper',
+        'js/controllers/cesium.layerCollection.controller'
+    ], function (Application, _, Marionette, Cesium, Q, wreqr, properties, CesiumMetacard, $, DrawHelper,
+                 LayerCollectionController) {
         "use strict";
 
-        var imageryProviderTypes = {
-            OSM: Cesium.OpenStreetMapImageryProvider,
-            AGM: Cesium.ArcGisMapServerImageryProvider,
-            BM: Cesium.BingMapsImageryProvider,
-            WMS: Cesium.WebMapServiceImageryProvider,
-            WMT: Cesium.WebMapTileServiceImageryProvider,
-            TMS: Cesium.TileMapServiceImageryProvider,
-            GE: Cesium.GoogleEarthImageryProvider,
-            CT: Cesium.CesiumTerrainProvider,
-            AGS: Cesium.ArcGisImageServerTerrainProvider,
-            VRW: Cesium.VRTheWorldTerrainProvider,
-            SI: Cesium.SingleTileImageryProvider
-        };
+        var imageryProviderTypes = LayerCollectionController.imageryProviderTypes;
+
+        var CesiumLayerCollectionController = LayerCollectionController.extend({
+            initialize: function () {
+                this.listenTo(wreqr.vent, 'layer:pref:reorder:bigMap', this.reIndexAll);
```

Comment:
To make it easier to track events, we try to have the first part of the event be the module that the event is thrown from. (This is going to change in later versions of the UI, but that is how it is for now.)